apisix/plugins/http-logger.lua

```diff
@@ -110,13 +113,6 @@ local function send_http_data(conf, log_message)
             .. "body[" .. httpc_res:read_body() .. "]"
     end
 
-    -- keep the connection alive
-    ok, err = httpc:set_keepalive(conf.keepalive)
```

Comment:
is it a default action ? although `keepalive` is not in the plugin's schema.

Reply:
the old code is wrong. that is why I delete them.

Reply:
if we want to call `set_keepalive`, need to define them first.